concent_api/utils/logging.py

```diff
@@ -285,3 +325,15 @@ def log_operation_validation_failed(
         subtask_id,
         client_key
     ))
+
+
+def _get_field_value_from_messages_for_logging(
+    field_name: MessageFields,
+    message: Message
+)->str:
+    try:
+        return_value = get_field_from_message(message, field_name.value) if message is not None else None
```

Comment:
1) it is unnecessary: you take `None` if message is `None` and one lien below you change it to `UNAVAILABLE`. Why not just:
```python
return get_field_from_message(message, field_name.value) if message is not None else 'UNAVAILABLE'
```

2) Why do you catch `AssertionError`? I don't think it is a good design... I guess we shouldn't get it in normal circumstances...

Reply:
I thought about it. I wanted to make this method 'safe', I agree we shouldn't get `AssertionError`, but if we get it's better to handle it in other place. In logger my only 'handling' is write 'unavailable'. But i wasn't sure if it good idea, changed with your suggestion